docs/release/support_matrix.md

```diff
@@ -1,7 +1,428 @@
+* [RHEL 8.7](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.7_release_notes/index)
+* [RHEL 8.8](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/index)
+
+Links to Rocky Linux 8 Release Notes:
+
+* [Rocky Linux 8.6](https://docs.rockylinux.org/release_notes/8_6/)
+* [Rocky Linux 8.7](https://docs.rockylinux.org/release_notes/8_7/)
+* [Rocky Linux 8.8](https://docs.rockylinux.org/release_notes/8_8/)
+
+Links to AlmaLinux 8 Release Notes:
+
+* [AlmaLinux 8.6](https://wiki.almalinux.org/release-notes/8.6.html)
+* [AlmaLinux 8.7](https://wiki.almalinux.org/release-notes/8.7.html)
+* [AlmaLinux 8.8](https://wiki.almalinux.org/release-notes/8.8.html)
+
+Refer to the [RHEL Life Cycle](https://access.redhat.com/support/policy/updates/errata/)
+description on the Red Hat support website for information on RHEL support phases.
+
+
+### Enterprise Linux 9 (EL9): RHEL 9, Rocky Linux 9, AlmaLinux 9
+
+DAOS Version 2.4.0 has not been validated and is not supported on EL9.
+It is a roadmap item to validate EL 9.2 (or later) with DAOS Version 2.6,
+and potentially with DAOS 2.4.1.
+
+Links to RHEL 9 Release Notes:
+
+* [RHEL 9.0](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.0_release_notes/index)
+* [RHEL 9.1](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.1_release_notes/index)
+* [RHEL 9.2](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.2_release_notes/index)
+
+Links to Rocky Linux Release Notes:
+
+* [Rocky Linux 9.0](https://docs.rockylinux.org/release_notes/9_0/)
+* [Rocky Linux 9.1](https://docs.rockylinux.org/release_notes/9_1/)
+* [Rocky Linux 9.2](https://docs.rockylinux.org/release_notes/9_2/)
+
+Links to AlmaLinux Release Notes:
+
+* [AlmaLinux 9.0](https://wiki.almalinux.org/release-notes/9.0.html)
+* [AlmaLinux 9.1](https://wiki.almalinux.org/release-notes/9.1.html)
+* [AlmaLinux 9.2](https://wiki.almalinux.org/release-notes/9.2.html)
+
+
+### Unsupported Linux Distributions
+
+With DAOS Version 2.4, CentOS 7 and RHEL 7 are no longer supported.
+Please update your DAOS servers to a supported EL8 level before updating to DAOS 2.4.
+
+DAOS also does not support
+openSUSE Tumbleweed,
+Fedora,
+CentOS Linux,
+CentOS Stream,
+Ubuntu, or
+Oracle Linux.
+
+
+## Operating Systems supported for DAOS Clients
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+In DAOS Version 2.4, the supported Linux distributions and versions for DAOS clients
+are identical to those for DAOS servers. Please refer to the previous section for details.
+
+In future DAOS releases, DAOS client support may be added for additional
+Linux distributions and/or versions.
+
+
+## Fabric Support
+
+DAOS Version 2.4 supports both OFI [libfabric](https://ofiwg.github.io/libfabric/).
+and UCF [UCX](https://openucx.org/) for communication in the DAOS data plane.
```

Comment:
```suggestion
and UCX [UCX](https://openucx.org/) for communication in the DAOS data plane.
```

Reply:
nope - UCF is the Foundation and UCX is the software ... like "OFI libfabric"